packages/components/legacy_modal/jest.setup.ts

```diff
@@ -0,0 +1,5 @@
+import ReactModal from "react-modal";
+import "@testing-library/jest-dom";
+import "jest-styled-components";
+
+jest.spyOn(ReactModal, "setAppElement").mockImplementation(() => undefined);
```

Comment:
If we are mocking the implementation of setAppElement entirely do we still need to call it from the test and wrap the fake parent element around the modal?

Reply:
Removing the parent element/call to setAppElement in `jest.setup` still allows the modal to render correctly. I think we can drop the wrapping element and call to `setAppElement`.